### PR TITLE
Revert klog fix

### PR DESCRIPTION
**Pull request by LeviHarrison**
After https://github.com/simonpasquier/klog-gokit/pull/18 klog-gokit v3 supports go-kit/log so we can revert the hotfix.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -48,7 +47,7 @@ require (
 	github.com/prometheus/client_golang v1.11.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.29.0
-	github.com/prometheus/exporter-toolkit v0.6.0
+	github.com/prometheus/exporter-toolkit v0.5.1
```
Comment (roidelapluie) [inline go.mod:50]: Downgrading ithub.com/prometheus/exporter-toolkit v0.6.0 ?

Comment (LeviHarrison) [reply go.mod:50]: Weird

**Comment by LeviHarrison** [comment]
Wait some stuff is broken

**Comment by LeviHarrison** [comment]
OK should be fixed.

**Comment by roidelapluie** [comment]
Thanks!

**Pull request merged**